### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by yqh984638220** [dmis-lab/GeNER#1]
Hello, may I ask how to generate the finer grained tags used in the paper, such as Conllo03 data set and person, which are more refined into athlete,politicion and actor?

**Comment by nowkim** [comment]
Hi @yqh984638220, Thanks for the question!

Please see the configuration file for CoNLL-2003 through the link below. In the JSON file, a "subtype" defines a fine-grained entity class. For example, you can see three subtypes "athlete", "politician", and "actor" for a single entity type "PER" in the JSON file. Note that "type" and "top_k" indicate a coarse-grained entity type and the number of sentences to retrieve, respectively.

-**Link**:  https://github.com/dmis-lab/GeNER/blob/main/configs/conll-2003.json

Feel free to ask me if you have any further questions

**Comment by yqh984638220** [comment]
Hello, I also want to ask about the use of DensePhreses. I have installed DensePhrase according to your link, is it installed under the directory of GeNER? Are the rest of the DensePhrases model and the phrase index also downloaded from this website? Is the training set, verification set, and test set also split according to the data set generated for the specific requirements? Or is it a separate json file?

**Comment by nowkim** [comment]
(1) DensePhrases can be installed in any path/directory. You can set "DENSEPHRASES_PATH" to load DensePhrases when running GeNER. (https://github.com/dmis-lab/GeNER#retrieval)

(2) You can download the models and index using the DensePhrases repository. (https://github.com/princeton-nlp/DensePhrases#installation)

(3) Synthetic datasets generated by GeNER are used only as training data. For the validation and test sets, we use human-annotated benchmark datasets. You can download the benchmark datasets from our repository. (https://github.com/dmis-lab/GeNER#ner-benchmarks)

**Comment by yqh984638220** [comment]
Hi, I was also wondering if you had a data set generated for conll03 (more granular with 9 entity tags), or if you had a model trained for conll03 (more granular with 9 entity tags). For some reason, the densephrases index couldn't be downloaded, I would appreciate it if you could provide me with both.

**Comment by nowkim** [comment]
Hi, we provide generated training datasets for the six main benchmarks we used, including CoNLL-2003. Please see the download link below.

- Data download: http://nlp.dmis.korea.edu/projects/gener-kim-et-al-2022/synthetic_datasets-221105.zip

We don't provide model checkpoints, but I think you can easily reproduce the best performance on CoNLL-2003 using the training data we provide. After you get the data, follow the instruction to train your model! (You may need to run the model multiple times with different random seeds to achieve exactly the same scores as reported in the paper.)

- https://github.com/dmis-lab/GeNER#self-training

**Comment by nowkim** [comment]
The download link does not seem to work when I directly click this. If you have the same problem, copy the link and paste it to the address bar. If you are using Linux, use "wget" command.

**Comment by yqh984638220** [comment]
Hello, I have downloaded the manually generated data set you provided and had a look. As for the generated conll03 training data, it seems that there are only three physical labels: PER,ORG and LOC. There are no more detailed physical labels, such as athlete, city and company. Can you provide a manually generated conll03 data set for such tags? Because the phrase index is too large, for some reason I was unable to download it, I would appreciate it if you could provide the data set of entity tags I described such as athlete, city, company.

**Comment by nowkim** [comment]
Hi, let me know your email. I'll create a dataset annotated with both fine-grained (e.g., athlete, politician, etc.) and coarse-grained entity types (e.g., PER, LOC, ORG), and send it to you!

Do you only need a training set for CoNLL-2003? Please let me know if you need other data as well!

**Comment by yqh984638220** [comment]
Yes, I only need the data set of ConLL03, my email address is 984638220@qq.com, thank you for providing.

**Comment by yqh984638220** [comment]
Hi, did you say last time to create a data set that includes athlete, politician labels? I have been waiting for your email these days. I would appreciate it if you could send it to me once it is finished.

**Comment by nowkim** [comment]
Hi, yes, I'll send it within a day

**Comment by yqh984638220** [comment]
Hi, I want to know that when I manually generated conll03 training set (labeled athlete, politician), validation and testing, I heard last time that you used the manually annotated benchmark data set, but did you use the validation and testing set of conll03 in fine-grained? Another puzzle is that the label of conll03 data set under benchmark is inconsistent with that of manually generated training set. Is this OK?

**Comment by nowkim** [comment]
The CoNLL-2003 dataset offers only general entity types, such as "PER", "LOC", and "ORG", and does not provide annotations for more specific subtypes. 

On the other hand, in the GeNER framework, synthetic data is generated using fine-grained entity types. However, during the annotation process, these fine-grained labels are mapped to their corresponding coarse-grained entity types. For instance, labels such as "athlete", "actor", and "politician" are mapped to the "PER" type. Likewise, other specific labels are mapped to either "LOC" or "ORG". This results in a final type set of ["PER", "LOC", "ORG"], which is identical to that of CoNLL-2003.

Please refer to our paper for details.
